src/py42/services/orgs.py

```diff
@@ -79,8 +103,8 @@ def get_page(self, page_num, page_size=None, **kwargs):
         Returns:
             :class:`py42.response.Py42Response`
         """
+        uri = "/api/v3/orgs"
```

Comment:
I don't think this new api implements paging, so I'll add some docs around that

Reply:
Does it just always return all results in a single request?

Reply:
yeah, that's my understanding